utility_network/display-content-of-utility-network-container/src/main/java/com/esri/samples/display_content_of_utility_network_container/DisplayContentOfUtilityNetworkContainerController.java

```diff
@@ -0,0 +1,305 @@
+
+      // create a utility network, add it to the map's collection of utility networks, and load it
+      utilityNetwork = new UtilityNetwork(featureServiceURL);
+      map.getUtilityNetworks().add(utilityNetwork);
+      utilityNetwork.addDoneLoadingListener(() -> {
+        // show an error if the utility network did not load
+        if (utilityNetwork.getLoadStatus() != LoadStatus.LOADED) {
+          new Alert(Alert.AlertType.ERROR, "Error loading Utility Network.").show();
+        }
+      });
+      utilityNetwork.loadAsync();
+
+      // hide the progress indicator once the map view draw status has completed
+      mapView.addDrawStatusChangedListener(listener -> {
+        if (listener.getDrawStatus() == DrawStatus.COMPLETED) {
+          progressIndicator.setVisible(false);
+        }
+      });
+
+      // set the map to the mapview and set the map view's viewpoint
+      mapView.setMap(map);
+      mapView.setViewpoint(new Viewpoint(41.801504, -88.163718, 4e3));
+
+    } catch (Exception e) {
+      e.printStackTrace();
+    }
+  }
+
+  /**
+   * Identifies the feature clicked on the map, gets their containment associations and elements, and displays the content
+   * of the elements features in a container view.
+   *
+   * @param e event registered when the map view is clicked on
+   */
+  @FXML
+  private void handleMapViewClicked(MouseEvent e) {
+
+    if (e.getButton() == MouseButton.PRIMARY && e.isStillSincePress()) {
+      // get the clicked map point
+      Point2D screenPoint = new Point2D(e.getX(), e.getY());
+
+      // identify the feature clicked on
+      ListenableFuture<List<IdentifyLayerResult>> identifyLayerResultsFuture =
+        mapView.identifyLayersAsync(screenPoint, 10, false);
+      identifyLayerResultsFuture.addDoneListener(() -> {
+        try {
+          // get the result of the query
+          List<IdentifyLayerResult> identifyLayerResults = identifyLayerResultsFuture.get();
+
+          // check that results have been returned
+          if (!identifyLayerResults.isEmpty()) {
+
+            identifyLayerResults.forEach(layerResult -> {
+              // check if the layer result is a subtype feature layer
+              if (layerResult.getLayerContent() instanceof SubtypeFeatureLayer) {
+                // loop through each sub layer result
+                layerResult.getSublayerResults().forEach(sublayerResult -> {
+                  // filter the sublayer result's elements to find the first one which is an ArcGIS feature
+                  sublayerResult.getElements().stream().filter(element -> element instanceof ArcGISFeature)
+                    .findFirst().ifPresent(geoElement -> selectedContainerFeature = (ArcGISFeature) geoElement);
+
+                });
+              }
+            });
+
+            // create a container element using the selected feature
+            UtilityElement containerElement = utilityNetwork.createElement(selectedContainerFeature);
```

Comment:
Is there a risk here that `selectedContainedFeature` is null if no results/features are found in the query? e.g. if lines 157/158 don't happen. There maybe isn't that risk with the dataset being used. 

I forced it and you basically get an IllegalArgumentException which isn't handled in that try/catch (only Interuppted/Execution exceptions on line 242)